{
  "checkpoints": [
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00001\",\n  \"config\": {\n    \"lr\": 0.00020443752355522158,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 2,\n    \"resize_to\": 384,\n    \"divide_into_four\": true\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.00020443752355522158,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 2,\n    \"resize_to\": 384,\n    \"divide_into_four\": true\n  },\n  \"experiment_tag\": \"1_batch_size=2,divide_into_four=True,loss_fn=bce,lr=0.00020444,resize_to=384\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871056.940064,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00001_1_batch_size=2,divide_into_four=True,loss_fn=bce,lr=0.00020444,resize_to=384_2021-07-21_14-37-36\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00001_1_batch_size=2,divide_into_four=True,loss_fn=bce,lr=0.00020444,resize_to=384_2021-07-21_14-37-36/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58428, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58428, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00008\",\n  \"config\": {\n    \"lr\": 0.0006226882434824387,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 3,\n    \"resize_to\": 384,\n    \"divide_into_four\": false\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.0006226882434824387,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 3,\n    \"resize_to\": 384,\n    \"divide_into_four\": false\n  },\n  \"experiment_tag\": \"8_batch_size=3,divide_into_four=False,loss_fn=bce,lr=0.00062269,resize_to=384\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871057.0042589,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00008_8_batch_size=3,divide_into_four=False,loss_fn=bce,lr=0.00062269,resize_to=384_2021-07-21_14-37-37\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00008_8_batch_size=3,divide_into_four=False,loss_fn=bce,lr=0.00062269,resize_to=384_2021-07-21_14-37-37/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58364, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58364, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00004\",\n  \"config\": {\n    \"lr\": 0.0022273916029543,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 4,\n    \"resize_to\": 384,\n    \"divide_into_four\": false\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.0022273916029543,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 4,\n    \"resize_to\": 384,\n    \"divide_into_four\": false\n  },\n  \"experiment_tag\": \"4_batch_size=4,divide_into_four=False,loss_fn=bce,lr=0.0022274,resize_to=384\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871056.9700215,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00004_4_batch_size=4,divide_into_four=False,loss_fn=bce,lr=0.0022274,resize_to=384_2021-07-21_14-37-36\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00004_4_batch_size=4,divide_into_four=False,loss_fn=bce,lr=0.0022274,resize_to=384_2021-07-21_14-37-36/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58384, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58384, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00003\",\n  \"config\": {\n    \"lr\": 0.00014502815251566954,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 1,\n    \"resize_to\": 192,\n    \"divide_into_four\": false\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.00014502815251566954,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 1,\n    \"resize_to\": 192,\n    \"divide_into_four\": false\n  },\n  \"experiment_tag\": \"3_batch_size=1,divide_into_four=False,loss_fn=bce,lr=0.00014503,resize_to=192\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871056.959826,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00003_3_batch_size=1,divide_into_four=False,loss_fn=bce,lr=0.00014503,resize_to=192_2021-07-21_14-37-36\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00003_3_batch_size=1,divide_into_four=False,loss_fn=bce,lr=0.00014503,resize_to=192_2021-07-21_14-37-36/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58426, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58426, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00009\",\n  \"config\": {\n    \"lr\": 0.0003989401270406467,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 2,\n    \"resize_to\": 192,\n    \"divide_into_four\": false\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.0003989401270406467,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 2,\n    \"resize_to\": 192,\n    \"divide_into_four\": false\n  },\n  \"experiment_tag\": \"9_batch_size=2,divide_into_four=False,loss_fn=bce,lr=0.00039894,resize_to=192\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871057.012819,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00009_9_batch_size=2,divide_into_four=False,loss_fn=bce,lr=0.00039894,resize_to=192_2021-07-21_14-37-37\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00009_9_batch_size=2,divide_into_four=False,loss_fn=bce,lr=0.00039894,resize_to=192_2021-07-21_14-37-37/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58417, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58417, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00005\",\n  \"config\": {\n    \"lr\": 0.0023726238464784524,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 3,\n    \"resize_to\": 384,\n    \"divide_into_four\": true\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.0023726238464784524,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 3,\n    \"resize_to\": 384,\n    \"divide_into_four\": true\n  },\n  \"experiment_tag\": \"5_batch_size=3,divide_into_four=True,loss_fn=bce,lr=0.0023726,resize_to=384\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871056.97896,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00005_5_batch_size=3,divide_into_four=True,loss_fn=bce,lr=0.0023726,resize_to=384_2021-07-21_14-37-36\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00005_5_batch_size=3,divide_into_four=True,loss_fn=bce,lr=0.0023726,resize_to=384_2021-07-21_14-37-36/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58405, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58405, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00007\",\n  \"config\": {\n    \"lr\": 0.03023187539245397,\n    \"loss_fn\": \"noise_robust_dice\",\n    \"batch_size\": 1,\n    \"resize_to\": 192,\n    \"divide_into_four\": false\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.03023187539245397,\n    \"loss_fn\": \"noise_robust_dice\",\n    \"batch_size\": 1,\n    \"resize_to\": 192,\n    \"divide_into_four\": false\n  },\n  \"experiment_tag\": \"7_batch_size=1,divide_into_four=False,loss_fn=noise_robust_dice,lr=0.030232,resize_to=192\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871056.996074,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00007_7_batch_size=1,divide_into_four=False,loss_fn=noise_robust_dice,lr=0.030232,resize_to=192_2021-07-21_14-37-36\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00007_7_batch_size=1,divide_into_four=False,loss_fn=noise_robust_dice,lr=0.030232,resize_to=192_2021-07-21_14-37-36/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58416, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58416, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00006\",\n  \"config\": {\n    \"lr\": 0.027843408044786137,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 1,\n    \"resize_to\": 384,\n    \"divide_into_four\": false\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.027843408044786137,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 1,\n    \"resize_to\": 384,\n    \"divide_into_four\": false\n  },\n  \"experiment_tag\": \"6_batch_size=1,divide_into_four=False,loss_fn=bce,lr=0.027843,resize_to=384\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871056.9875658,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00006_6_batch_size=1,divide_into_four=False,loss_fn=bce,lr=0.027843,resize_to=384_2021-07-21_14-37-36\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00006_6_batch_size=1,divide_into_four=False,loss_fn=bce,lr=0.027843,resize_to=384_2021-07-21_14-37-36/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58363, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58363, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00000\",\n  \"config\": {\n    \"lr\": 0.0003404304792789932,\n    \"loss_fn\": \"noise_robust_dice\",\n    \"batch_size\": 3,\n    \"resize_to\": 384,\n    \"divide_into_four\": false\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.0003404304792789932,\n    \"loss_fn\": \"noise_robust_dice\",\n    \"batch_size\": 3,\n    \"resize_to\": 384,\n    \"divide_into_four\": false\n  },\n  \"experiment_tag\": \"0_batch_size=3,divide_into_four=False,loss_fn=noise_robust_dice,lr=0.00034043,resize_to=384\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871056.926329,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00000_0_batch_size=3,divide_into_four=False,loss_fn=noise_robust_dice,lr=0.00034043,resize_to=384_2021-07-21_14-37-36\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00000_0_batch_size=3,divide_into_four=False,loss_fn=noise_robust_dice,lr=0.00034043,resize_to=384_2021-07-21_14-37-36/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58414, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58414, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"trainable_name\": \"train_segmentation\",\n  \"trial_id\": \"6dcb1_00002\",\n  \"config\": {\n    \"lr\": 0.007271286701886563,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 1,\n    \"resize_to\": 384,\n    \"divide_into_four\": true\n  },\n  \"local_dir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36\",\n  \"evaluated_params\": {\n    \"lr\": 0.007271286701886563,\n    \"loss_fn\": \"bce\",\n    \"batch_size\": 1,\n    \"resize_to\": 384,\n    \"divide_into_four\": true\n  },\n  \"experiment_tag\": \"2_batch_size=1,divide_into_four=True,loss_fn=bce,lr=0.0072713,resize_to=384\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"resources\": {\n    \"cpu\": 1,\n    \"gpu\": 0,\n    \"memory\": 0,\n    \"object_store_memory\": 0,\n    \"extra_cpu\": 0,\n    \"extra_gpu\": 0,\n    \"extra_memory\": 0,\n    \"extra_object_store_memory\": 0,\n    \"custom_resources\": {},\n    \"extra_custom_resources\": {}\n  },\n  \"placement_group_factory\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"800595b2000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c085f62756e646c6573945d947d94288c03435055944b018c03475055944b008c066d656d6f7279944b008c136f626a6563745f73746f72655f6d656d6f7279944b0075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\"\n  },\n  \"stopping_criterion\": {},\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"last_result\": {},\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1626871056.9506292,\n  \"logdir\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00002_2_batch_size=1,divide_into_four=True,loss_fn=bce,lr=0.0072713,resize_to=384_2021-07-21_14-37-36\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/train_segmentation_6dcb1_00002_2_batch_size=1,divide_into_four=True,loss_fn=bce,lr=0.0072713,resize_to=384_2021-07-21_14-37-36/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 718, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 688, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 62, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/worker.py\\\", line 1495, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58415, ip=10.204.4.69)\\n  File \\\"python/ray/_raylet.pyx\\\", line 501, in ray._raylet.execute_task\\n  File \\\"python/ray/_raylet.pyx\\\", line 451, in ray._raylet.execute_task.function_executor\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/_private/function_manager.py\\\", line 563, in actor_method_executor\\n    return method(__ray_actor, *args, **kwargs)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 173, in train_buffered\\n    result = self.train()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 232, in train\\n    result = self.step()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 366, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 512, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=58415, ip=10.204.4.69)\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 248, in run\\n    self._entrypoint()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 315, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 580, in _trainable_func\\n    output = fn()\\n  File \\\"/cluster/home/samuelbe/.local/lib/python3.8/site-packages/ray/tune/utils/trainable.py\\\", line 331, in inner\\n    trainable(config, **fn_kwargs)\\n  File \\\"tune_hyperparameters.py\\\", line 20, in train_segmentation\\n    trainer.fit(model, dm)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 458, in fit\\n    self._run(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/trainer.py\\\", line 715, in _run\\n    self.accelerator.setup(self, model)  # note: this sets up self.lightning_module\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/cpu.py\\\", line 39, in setup\\n    return super().setup(trainer, model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 92, in setup\\n    self.setup_optimizers(trainer)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/accelerators/accelerator.py\\\", line 374, in setup_optimizers\\n    optimizers, lr_schedulers, optimizer_frequencies = self.training_type_plugin.init_optimizers(\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/plugins/training_type/training_type_plugin.py\\\", line 190, in init_optimizers\\n    return trainer.init_optimizers(model)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/pytorch_lightning/trainer/optimizers.py\\\", line 34, in init_optimizers\\n    optim_conf = model.configure_optimizers()\\n  File \\\"/cluster/home/samuelbe/new/rosas_road_segmentation/common/lightning/base.py\\\", line 75, in configure_optimizers\\n    return torch.optim.Adam(self.parameters(), lr=self.lr)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/adam.py\\\", line 48, in __init__\\n    super(Adam, self).__init__(params, defaults)\\n  File \\\"/cluster/apps/python/3.8.5_gpu_gcc820/x86_64/lib64/python3.8/site-packages/torch/optim/optimizer.py\\\", line 49, in __init__\\n    raise ValueError(\\\"optimizer got an empty parameter list\\\")\\nValueError: optimizer got an empty parameter list\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}"
  ],
  "runner_data": {
    "_max_pending_trials": 79,
    "_metric": null,
    "_total_time": 0,
    "_iteration": 28,
    "_has_errored": true,
    "_fail_fast": false,
    "_server_port": null,
    "_cached_trial_decisions": {},
    "_queued_trial_decisions": {},
    "_updated_queue": true,
    "_should_stop_experiment": false,
    "_local_checkpoint_dir": "/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36",
    "_remote_checkpoint_dir": null,
    "_stopper": {
      "_type": "CLOUDPICKLE_FALLBACK",
      "value": "80059527000000000000008c107261792e74756e652e73746f70706572948c0b4e6f6f7053746f707065729493942981942e"
    },
    "_resumed": false,
    "_start_time": 1626871056.7581892,
    "_last_checkpoint_time": -Infinity,
    "_session_str": "2021-07-21_14-37-36",
    "checkpoint_file": "/cluster/home/samuelbe/ray_results/train_segmentation_2021-07-21_14-37-36/experiment_state-2021-07-21_14-37-36.json",
    "_checkpoint_period": "auto",
    "launch_web_server": false
  },
  "stats": {
    "start_time": 1626871056.7581892,
    "timestamp": 1626871056.907385
  }
}